src/data.ts

```diff
@@ -17,3 +17,5 @@ export default {
     twitter: 'SealCredNFT',
   },
 }
+
+export type DataContractNames = keyof typeof data
```

Comment:
Maybe we should put it in the `models`? As far as I know, we use it separately from `data` in some places

Reply:
Okay, let's keep it the same